state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -262,394 +306,168 @@ def reinit(self, dev):
-
-            # the new log file has number n + 1; n is the current
-            # greatest file number
-            self.logfile = "/sd/logs/log" + "{:03}".format(n + 1) + ".txt"
-
-            # open the new logfile and write the time it was created +
-            # the current status
-            with open(self.logfile, "a") as log:
-                log.write('# Created: {:.0f}\r\n# Status: {}\r\n'.format(
-                    time.monotonic(), self.status))
-
-            # print a confirmation message that a new logfile was created
-            print('New log file:', self.logfile)
-
-    def print_file(self, filedir=None):
-        """
-        print a file given its directory; file directory is by default None
-        """
-
-        # if no file directory is passed, use the directory of the log file
-        if filedir is None:
-            filedir = self.logfile
-
-        print('--- Printing File: {} ---'.format(filedir))
-
-        # open the current file directory as read only, print line by line
-        # (removing whitespace)
-        with open(filedir, "r") as file:
-            for line in file:
-                print(line.strip())
-
-    def send_file(self, c_size, send_buffer, filename):
-        """
-        send a file given character size, buffer size, and the filename
-        """
-
-        # number of packets is the size of the filename / character size
-        num_packets = int(stat(filename)[6] / c_size)
-
-        # open the file
-        with open(filename, "rb") as f:
-            # for each packet
-            for i in range(num_packets + 1):
-                # move the cursor to the end of i * character size,
-                # add to buffer
-                f.seek(i * c_size)
-                f.readinto(send_buffer)
-
-                # return bytes; yield keyword returns without destroying
-                # states of local vars
-                yield bytes([i, 0x45, num_packets])
-
-    def save(self, dataset, savefile=None):
-        """
-        save the passed dataset to the passed savefile
-        dataset should be a set of lists; each line is a list:
-            save(([line1],[line2]))
-        to save a string, make it an item in a list:
-            save(['This is my string'])
-        by default, savefile is not passed
-        """
-        # if no savefile is passed, use the current filename attribute
-        # by default
-        if savefile is None:
-            savefile = self.filename
-
-        # open save file
+
+# initialize Satellite as cubesat
+cubesat = Satellite()
+
+"""
+IMU-related functions
+TODO: cubesat.imu hardware check
+"""
+
+def acceleration():
+    """ return the accelerometer reading from the IMU """
+    return cubesat.IMU.accel
+
+def magnetic():
+    """ return the magnetometer reading from the IMU """
+    return cubesat.IMU.mag
+
+def gyro():
+    """ return the gyroscope reading from the IMU """
+    return cubesat.IMU.gyro
+
+def temperature():
```

Comment:
What if we rename this to `temperature_imu()`?

Reply:
I like this, we also don't use this anywhere anyway so we can totally just rename it to whatever we want

Reply:
I think if we're not using it anywhere currently, it's an easy rename.